modules/ROOT/partials/depl-examples/ubuntu-compose/shared-setup.adoc

```diff
@@ -573,16 +575,91 @@ docker volume ls
 docker volume rm {ocis_wopi}_ocis-config {ocis_wopi}_ocis-data
 ----
 
-== Updating
+== Updating and Upgrading
+
+=== Updating
+
+// for testing
+//:version-type: production
 
 ifeval::["{version-type}" == "rolling"]
-Note that this deploymment can, for the time being, only be updated within Infinite Scale rolling releases.
+For rolling releases, the following steps are essential to avoid breaking the setup. This is because rolling releases depend on consecutive updates to be taken.
```

Comment:
```suggestion
For rolling releases, the following steps are essential to avoid breaking the setup. This is because rolling releases depend on all updates being done consecutively.
```